src/components/MenuItem/MenuItem.stories.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { MenuItem } from "./MenuItem";
+import { Icons } from "./../../icons/Icons";
+
+export default {
+    component: MenuItem,
+    title: "Components/MenuItem",
+};
+
+export const Default = () => {
+    return (
+        <div style={{ width: "375px" }}>
+            <MenuItem
+                IconElement={Icons.CROSS_CIRCLE}
+                itemTitle="goats cheese romana"
+                itemPrice="£12.50"
```

Comment:
You dont need an `item` prefix to these attributes, just `title` and `price` will do. We know they are the MenuItem price and title by the very virtue of them being attributes on the comp. 

When names are written like this it can confuse a person into thinking that there is something they are missing.